mayastor/src/bdev/nexus/nexus_child.rs

```diff
@@ -459,6 +464,9 @@ impl NexusChild {
             state: AtomicCell::new(ChildState::Init),
             prev_state: AtomicCell::new(ChildState::Init),
             remove_channel: mpsc::channel(0),
+            guid: Guid::from_str("00000000-0000-0000-0000-000000000000")
```

Comment:
If we're just setting itt to the same thing every time can we not print it? (Why do we have this?)

Reply:
This is needed to identify the child in a manner that is persistent across restarts. The proper value is "initialised" from the metatadata in the MayaMeta partition, but it is too early to do that here. It just needs to be initialised to something. However have now changed to initialise it to uuid::Uuid::nil()